content/benefits/claims/release-notes/2024-06-18.md

```diff
@@ -0,0 +1,9 @@
+We added an optional request attribute for internal VA consumers to include 'claimNotes' on a disability compensation claim.
+
+This release includes updates to the following v2 endpoints:
```

Comment:
Is this request attribute (claimNotes) affecting these three endpoints? What is the relationship between adding 'claimNotes' and these endpoints?

Reply:
@joeabraun Each of the 3 endpoints will now accept the attribute `claimNotes` because they all use the same schema.

Reply:
Hi @joeabraun - Are you looking for more context around how the `claimNotes` will be used for each service? The 3 endpoints noted are used to generate and/or validate the 526EZ PDF for disability compensation claims. We will accept up to 4k characters that will be added to an additional page on the 526EZ PDF.

Reply:
Hi @stiehlrod and @klwatanabe - yes, I just wanted some context how claimNotes was being used for each endpoint. It might be helpful to our users who now see 'claimNotes'  added to these endpoints to understand what it is/does